crates/http/src/wagi.rs

```diff
@@ -143,7 +143,13 @@ impl HttpExecutor for WagiHttpExecutor {
             .or_else(ignore_successful_proc_exit_trap)?;
         tracing::info!("Module execution complete");
 
-        util::compose_response(&stdout_buffer.take())
+        let stdout = stdout_buffer.take();
+        ensure!(
+            !stdout.is_empty(),
+            "nothing was written to stdout; are you using a WAGI component?"
```

Comment:
Could this be a bit more specific re: executors?

Reply:
You mean like "this component is using the WAGI executor but nothing was written to stdout"?

Reply:
Yes — specifically,it should guide me to look at my `executor` field in `spin.toml`.